DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -43,17 +43,17 @@
     <header class="nhsuk-header nhsuk-header--organisation nhsuk-header--white" role="banner">
       <div class="nhsuk-width-container nhsuk-header__container">
         <div class="nhsuk-header__logo">
-          <div class="nhsuk-header__link--service">
-            <a class="nhsuk-header__link" href="@Configuration["CurrentSystemBaseUrl"]/Home?action=appselect" aria-label="NHS Digital Learning Solutions @(ViewData["Application"] ?? "") - Switch Application">
-              <svg class="nhsuk-logo" xmlns="http://www.w3.org/2000/svg" role="presentation" focusable="false" viewBox="0 0 40 16">
-                <path class="nhsuk-logo__background" d="M0 0h40v16H0z"></path>
-                <path class="nhsuk-logo__text" d="M3.9 1.5h4.4l2.6 9h.1l1.8-9h3.3l-2.8 13H9l-2.7-9h-.1l-1.8 9H1.1M17.3 1.5h3.6l-1 4.9h4L25 1.5h3.5l-2.7 13h-3.5l1.1-5.6h-4.1l-1.2 5.6h-3.4M37.7 4.4c-.7-.3-1.6-.6-2.9-.6-1.4 0-2.5.2-2.5 1.3 0 1.8 5.1 1.2 5.1 5.1 0 3.6-3.3 4.5-6.4 4.5-1.3 0-2.9-.3-4-.7l.8-2.7c.7.4 2.1.7 3.2.7s2.8-.2 2.8-1.5c0-2.1-5.1-1.3-5.1-5 0-3.4 2.9-4.4 5.8-4.4 1.6 0 3.1.2 4 .6"></path>
-                <image src="https://assets.nhs.uk/images/nhs-logo.png" xlink:href=""></image>
-              </svg>
-            </a>
-
-            <span class="nhsuk-header__transactional-service-name">Digital Learning Solutions @(ViewData["Application"] ?? "")</span>
-          </div>
+          <a class="nhsuk-header__link nhsuk-header__link--service"
+             href="@Configuration["CurrentSystemBaseUrl"]/@(ViewData["HeaderPath"] ?? "Home?action=appselect")"

```

Comment:
This won't work since the current system base url isn't the base url for our application, so it won't work for redirecting to the current courses. I think it would be easiest to pass in the whole url in the view data. You can use appRootPath for the base url for our application 🙂

Reply:
I couldn't find where the appRootPath was. I ended up just using relative paths (eg. `href="/LearningPortal/Current"`). Is this ok? If not, where should I look to find the appRootPath? 🙂

Reply:
As long as that works then that's fine! The appRootPath is also in the appsettings, like the currentSystemBaseUrl. But your solutions is simpler so happy to go with that as long as it works 🙂